library/xml

```diff
@@ -106,6 +106,12 @@ options:
     choices:
       - "yaml"
       - "xml"
+  strip_tags:
+    description:
+      - "Remove CDATA tags from XML"
```

Comment:
So I don't understand that description. The documentation says: *strip_cdata - replace CDATA sections by normal text content (default: True)* but that's not completely clear for me either.

I would prefer to know exactly what this parameter does, and I also prefer if that parameter is renamed to `strip_cdata` so that there is a clear correlation between the option and its low-level call.